### PR TITLE
add fixer for namespace immediately following opening tag

### PR DESCRIPTION
**Pull request by benharold**
This fixer will remove any content between the opening tag and namespace declaration except for a single space character.

### CODE REVIEW

File: Symfony/CS/Fixer/Contrib/NamespaceFollowsOpeningTagFixer.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        // ignore non-monolithic files
+        if (!$tokens->isMonolithicPhp()) {
+            return $content;
+        }
+
+        // ignore files with short open tag
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/NamespaceFollowsOpeningTagFixer.php]: why?

File: Symfony/CS/Tests/Fixer/Contrib/NamespaceFollowsOpeningTagFixerTest.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        $this->makeTest($expected, $input);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideExamples()
+    {
+        return array(
+            array('<?php namespace X;'),
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Contrib/NamespaceFollowsOpeningTagFixerTest.php]: please add case with `<?` and `<?=`

**Comment by keradus** [comment]
I see that hhvm will be your friend ;)

File: Symfony/CS/Tests/Fixer/Contrib/NamespaceFollowsOpeningTagFixerTest.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            array('<?= "output"; ?>'),
+            array('<?php namespace X;', "<?php\n\n\n\nnamespace X;"),
+            array('<?php namespace X;', "<?php\r\n\r\n\r\n\r\nnamespace X;"),
+        );
+    }
+
+    public function testFixExampleWithComment()
+    {
+        $expected = <<<'EOF'
+<?php namespace Symfony\CS\Fixer\Contrib;
```
Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Contrib/NamespaceFollowsOpeningTagFixerTest.php]: Why remove the comment? Maybe it is better to move the `namespace` and leave the comment as is?

**Comment by keradus** [comment]
No community interests in this rule?
Also, I see a big conflict with DeclareStrictTypesFixer

**Comment by GrahamCampbell** [comment]
:-1: on this.

**Comment by keradus** [comment]
Thank you @benharold 
Closing due to community reaction.
If really wanted it could be done as part of #2062

**Pull request closed without merging**